### ISSUE TITLE
Create option to disable SI prefixes

### ISSUE THREAD

**Issue by albyr** [benyohaiphysics/FlamingoPhysics#1]
Many pupils struggle with, or are "frightened" by, the use of SI prefixes. It would be useful to have a toggle that would disable them in a given output.

**Comment by benyohaiphysics** [comment]
Good idea. I create the variables randomly using `sample(0:100,1)*10^(sample(-3:6,1))`. If you remove everything following the asterisk, it should achieve what you want.

I'm still trying to work out the best way to separate out all the drawing logic and adding in things like this as variables.

**Comment by albyr** [comment]
I think a simple ``usePrefixes = TRUE`` in the head and an ``if ... else`` statement would probably be the easiest way to go.
